ui/v2.5/src/components/Tagger/scenes/TaggerScene.tsx

```diff
@@ -186,6 +192,27 @@ export const TaggerScene: React.FC<PropsWithChildren<ITaggerScene>> = ({
     }
   }
 
+  function maybeRenderSpriteIcon() {
+    if (scene.paths.sprite && scene.paths.sprite.length > 0) {
+      // Even if a scene doesn't have a sprite generated, the path will be
+      // http://localhost:9999/scene/_sprite.jpg so check for that case
+      const spriteParts = scene.paths.sprite.split("/");
+      const filename = spriteParts.pop();
+      if (filename?.includes("_sprite") && filename.split("_")[0].length > 0) {
```

Comment:
Can this not be changed to check for `scene.files.length`?

Reply:
Originally was trying to not show it if the sprite wasn't yet generated, but couldn't really figure it out. `scene.files.length` is a better suggestion.